tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -270,13 +306,31 @@ def _create_representations(
         num_iterations = len(image_info)
         progress_reporter = PrintProgressReporter(print_interval=progress_interval)
         progress_reporter.reset(num_iterations, 'annotations')
+        segmentation_colors = (
+            (0, 0, 0), (128, 0, 0), (0, 128, 0), (128, 128, 0),
+            (0, 0, 128), (128, 0, 128), (0, 128, 128), (128, 128, 128),
+            (64, 0, 0), (192, 0, 0), (64, 128, 0), (192, 128, 0),
+            (64, 0, 128), (192, 0, 128), (64, 128, 128), (192, 128, 128),
+            (0, 64, 0), (128, 64, 0), (0, 192, 0), (128, 192, 0),
+            (0, 64, 128))
+        palette = np.asarray(segmentation_colors, dtype=np.uint8).reshape([21 * 3,])
+
 
         for (image_id, image) in enumerate(image_info):
             image_labels, _, _, _, _, is_crowd, segmentations = self._read_image_annotation(
                 image, annotations,
                 label_id_to_label
             )
-            annotation = CoCoInstanceSegmentationAnnotation(image[1], segmentations, image_labels)
+
+            if not self.semantic_only:
+                annotation = CoCoInstanceSegmentationAnnotation(image[1], segmentations, image_labels)
+            else:
+                # print(image_id)
+                h, w, _ = image[2]
+                mask_file = self.masks_dir / "{:012}.png".format(image[0])
+                self.make_mask(h, w, mask_file, image_labels, segmentations, palette)
+                annotation = SegmentationAnnotation(image[1], mask_file, mask_loader=GTMaskLoader.SCIPY)
```

Comment:
please, do not forget that serialization of path is platform specific thing. You need to wrap mask_file to string and made relative

Reply:
Fixed